databases/week1/homework.sql

```diff
@@ -0,0 +1,87 @@
+-- Find out how many tasks are in the task table
+select
+    COUNT(id)
+FROM
+    task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT
+    COUNT(id)
+from
+    task
+WHERE
+    due_date is Null;
+
+-- Find all the tasks that are marked as done
+SELECT
+    *
+FROM
+    task
+WHERE
+    status_id = "Done";
+
+-- Find all the tasks that are not marked as done
+SELECT
+    *
+FROM
+    task
+WHERE
+    status_id != "Done";
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT
+    *
+FROM
+    task
+ORDER BY
+    created ASC;
```

Comment:
The most recently would require you to order by DESC. Newer time = higher number